docs/whats-new/staging-release-notes.md

```diff
@@ -1,5 +1,19 @@
 # Staging release notes
 
+## March 2024
+
+**01 March 2024**
+
+Frontend version: 1.0.0-20240125.1442 | Backend version: 1.88.3-20240131.1209
+
+| Type   | Change  | Description |
+| ------ | ------- | ----------- |
+| **Feature** | New Automation APIs for app role management | We have introduced new Automation APIs allowing tenants to autonomously manage their app roles, enhancing efficiency and autonomy in role management. This feature is currently available through the Automation API, with plans to integrate it into the TechPass portal in the future. |
+| **Change** | Autonomous app role assignments for TechBiz-integrated applications | Tenants can now manage app role assignments for TechBiz-integrated applications autonomously, excluding TechBiz group assignments, through the Automation API. This change grants tenants greater control and flexibility over their app role assignments. |
```

Comment:
can we highlight that this change is rolled out for automation api only & that works is still undergoing to allow tenants to manage the app role assigments for techbiz-integrated apps on the portal